pkgs/development/lua-modules/generated-packages.nix

```diff
@@ -2063,14 +2065,14 @@ buildLuarocksPackage {
 luarocks-build-treesitter-parser = callPackage({ buildLuarocksPackage, fetchurl, fetchzip, luaOlder, luafilesystem }:
 buildLuarocksPackage {
   pname = "luarocks-build-treesitter-parser";
-  version = "4.1.0-1";
+  version = "5.0.0-1";
```

Comment:
This is what I was referring to. 
tree-sitter-norg depends on luarocks-build-treesitter-parser.

Reply:
Thanks, it works indeed.
I have pushed the revert.